ios/CodePush/CodePushUpdateUtils.m

```diff
@@ -3,6 +3,10 @@
 
 @implementation CodePushUpdateUtils
 
+NSString * const ManifestFolderPrefix = @"CodePush";
+NSString * const DefaultJsBundleName = @"main.jsbundle";
+NSString * const DefaultAssetsFolderName = @"assets";
```

Comment:
Isn't the "assets" folder name static/fixed? If so, the name "Default" is maybe a little misleading.